website/docs/about/accessibility-statement.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Accessibility statement
 navigation:
-  order: 105
+order: 105
```

Comment:
This needs to remain nested as it's a subitem of navigation (which contains other items, such as `hidden` or `label`)